src/widgets/vitals/vitals-form.component.tsx

```diff
@@ -27,8 +27,12 @@ export default function VitalsForm(props: vitalsFormProp) {
   const [temperature, setTemperature] = useState(null);
   const [weight, setWeight] = useState(null);
   const [height, setHeight] = useState(null);
-  const [dateRecorded, setDateRecorded] = useState(null);
-  const [timeRecorded, setTimeRecorded] = useState(null);
+  const [dateRecorded, setDateRecorded] = useState(
+    dayjs(new Date()).format("YYYY-MM-DD")
```

Comment:
I saw some places use `"YYYY-MMM-DD"` format.

Is there any standard format?

Reply:
yeap, [Figma designs](https://www.figma.com/file/YyVhWTNp89bNiOGadYrHBk/Patient-Chart---Master-(current-file)?node-id=3931%3A3810) at the moment using the default HTML input date which does't support the design standard.